### DISCUSSION TITLE
Import found subdomains as an input to other modules.

### DISCUSSION THREAD

**Discussion by dhayalandeena10** [blacklanternsecurity/bbot#1063] [General]
Hello team,

I have done subdomain enumeration already if i want to run httpx on them how can i do that please let me know

Best Regards.

**Comment by Sh4d0wHunt3rX** [comment]
Try this:
bbot -m httpx -t subdomains.txt

**Reply by TheTechromancer** [reply]
The above will work. However if you can, it's better to run httpx along with the subdomain enumeration (it is already included in `-f subdomain enum`). The reason is because BBOT is recursive, you will miss out on subdomains if you separate the passive and active phases. `httpx` itself can uncover new subdomains which when consumed by the subdomain enumeration modules can lead to the discovery of even more subdomains, etc.
